src/app/pages/home/asset/options-menu/options-menu.component.html

```diff
@@ -2,4 +2,11 @@
   <a mat-list-item (click)="openLink(options.Delete)">
     <span mat-line>{{ t('delete') }}</span>
   </a>
+  <a
```

Comment:
This `OptionsMenuComponent` should __NOT__ be reused for other pages or components (e.g. `PostCaptureCardComponent`). You should create another component for `PostCaptureCardComponent`.